docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -15,51 +15,108 @@ in the [Working with command-line compiler](command-line.md) tutorial.
 
 Kotlin compilers have a number of options for tailoring the compiling process.
 
-Using a build script, you can specify additional compilation options. Use the `compilerOptions` property of a Kotlin compilation task for it. 
-For example:
+When you use the Kotlin Gradle plugin in your project, the Gradle DSL allows comprehensive configuration of compiler options. 
+The Gradle DSL is available for [Kotlin Multiplatform](multiplatform-dsl-reference.md) and [JVM/Android](#target-the-jvm) projects.
 
-<tabs group="build-script">
-<tab title="Kotlin" group-key="kotlin">
+With the Gradle DSL, you can configure compiler options within the build script at three levels: 
+* **Extension level,** as a top-level block. These compiler options are common for all targets and shared source sets.
```

Comment:
Strictly speaking, only `kotlin {}` is a top-level block here, so I would avoid this term to not have to say things like "top-level within something".